python/railjson_generator/railjson_generator/infra_builder.py

```diff
@@ -30,57 +30,30 @@ def add_track_section(self, *args, **kwargs):
         self.infra.track_sections.append(track)
         return track
 
-    def add_point_switch(
-        self,
-        base: TrackEndpoint,
-        left: TrackEndpoint,
-        right: TrackEndpoint,
-        signal_on_ports: Mapping[str, Tuple[str, str]] = None,
-        **kwargs
-    ):
+    def add_point_switch(self, base: TrackEndpoint, left: TrackEndpoint, right: TrackEndpoint, **kwargs):
         """
         Adds a switch as well as all links between concerned track sections.
         """
         switch = PointSwitch(base=base, left=left, right=right, **kwargs)
-        l1 = self.add_link(base, left)
```

Comment:
This function created a link but also registered the two endpoint as neighbors
Now they no longer are neighbors so railjson-generator's pathfinding algorithm will be broken I believe
Is this intended ?

Reply:
it wasn't, thanks :)